src/classes/FoundBugs.js

```diff
@@ -50,7 +51,8 @@ class FoundBugs {
 	 *				sources: [{
 	 *					src: string,
 	 *					blocked: boolean,
-	 *					type: string
+	 *					type: string,
+	 *					request_id: string
```

Comment:
React was throwing an error in `app/panel/components/Blocking/Tracker.jsx` because we were using the `index` of a loop as the `key` and React doesn't like that.  There was nothing else on the tracker source that I could use as the `key` so I passed `request_id` so it could be my unique `key`.

Reply:
I see, thank you